README.md

```diff
@@ -1,17 +1,8 @@
 # Deis Monitor v2
-
-[![Build Status](https://travis-ci.org/deis/monitor.svg?branch=master)](https://travis-ci.org/deis/monitor)
-[![Docker Repository on Quay](https://quay.io/repository/deisci/monitor/status "Docker Repository on Quay")](https://quay.io/repository/deisci/monitor)
```

Comment:
that would be good. most people going to the repo aren't going to go into each subdirectory for image status. Having them all in the main README would be beneficial.